tools/packaging/verify-deb-install.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# Install tempo and check it's running
+dpkg -i ./tempo_*_linux_amd64.deb
```

Comment:
these 2 verify scripts are no longer used. they used to be referenced by release.yml as a testing step. do we want to restore the verify? they may require modifications to accept an unsigned package.

i'm good on just removing if we have high confidence in the packages.